admin/package.json

```diff
@@ -19,19 +19,19 @@
     "bikram-sambat-bootstrap": "^1.4.2",
     "bootstrap": "^3.4.1",
     "font-awesome": "^4.7.0",
-    "google-libphonenumber": "^3.2.2",
+    "google-libphonenumber": "^3.2.5",
     "gsm": "^0.1.4",
     "jquery": "^3.4.1",
-    "lodash": "^4.17.11",
-    "messageformat": "^2.2.1",
+    "lodash": "^4.17.15",
+    "messageformat": "^2.3.0",
     "moment": "^2.24.0",
-    "mustache": "^3.0.1",
-    "ng-redux": "^4.4.0",
+    "mustache": "^3.1.0",
+    "ng-redux": "^4.4.1",
     "object-path": "^0.11.4",
     "pouchdb-browser": "^7.1.1",
     "properties": "^1.2.1",
-    "redux": "^4.0.1",
-    "select2": "^4.0.3",
+    "redux": "^4.0.4",
+    "select2": "^4.0.11",
```

Comment:
Looks like this got bumped to latest. It works though!

Reply:
Yeah admin doesn't seem to have an issue. It's likely that something else is pulling in a select2 dependency (my prime suspect is enketo-core) which doesn't affect admin at all.